Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -1051,487 +1150,490 @@ public string keyboardLayout
         /// <summary>
         /// A synthetic button control that is considered pressed if any key on the keyboard is pressed.
         /// </summary>
-        /// <value>Control representing the synthetic "anyKey".</value>
+        /// <remarks><see cref="AnyKeyControl"/> representing the synthetic "anyKey".</remarks>
         public AnyKeyControl anyKey { get; protected set; }
 
         /// <summary>
-        /// The space bar key.
+        /// The space bar key at the bottom of the keyboard.
         /// </summary>
-        /// <value>Control representing the space bar key.</value>
+        /// <remarks><see cref="KeyControl"/> representing the space bar key.</remarks>
         public KeyControl spaceKey => this[Key.Space];
 
         /// <summary>
         /// The enter/return key in the main key block.
         /// </summary>
-        /// <value>Control representing the enter key.</value>
         /// <remarks>
+        /// <see cref="KeyControl"/> representing the enter key.
         /// This key is distinct from the enter key on the numpad which is <see cref="numpadEnterKey"/>.
         /// </remarks>
         public KeyControl enterKey => this[Key.Enter];
 
         /// <summary>
-        /// The tab key.
+        /// The tab key, located on the left side above the <see cref="capsLockKey"/>.
```

Comment:
Nitpick: tab keys are not always above capsLockKey. Maybe change it to 
```suggestion
        /// The tab key, which is generally located on the left side above the <see cref="capsLockKey"/>.
```